homeassistant/components/rki_covid/__init__.py

```diff
@@ -0,0 +1,172 @@
+"""RKI Covid numbers integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import aiohttp
+import async_timeout
+from rki_covid_parser.parser import RkiCovidParser
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client, update_coordinator
+
+from .const import DOMAIN
+from .data import DistrictData, StateData
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = ["sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict) -> bool:
+    """Set up the component into HomeAssistant."""
+    _LOGGER.debug("setup component.")
+    parser = RkiCovidParser(aiohttp_client.async_get_clientsession(hass))
+
+    # Make sure coordinator is initialized.
+    await get_coordinator(hass, parser)
+
+    # Return boolean to indicate that initialization was successful.
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up component from a config entry."""
+    _LOGGER.debug(f"Setup item from config entry: {entry.data}.")
+    # Forward the setup to the sensor platform.
```

Comment:
```suggestion
```

Reply:
Since it is a verbose|debug log, it should not matter to stay inside to add logging if necessary

Reply:
True but for me a quite pointless debugging in production code